### PR TITLE
Add note on X devices being unupported

### LINKED ISSUE #35508: New version of Tado doesn´t support API
### Feedback

The page about the Tado integration is exclusively for the V1, V2, V3/V3+ versions. The new 'Tado X' can now only be integrated through Matter. There is no API for Tado X.

### URL

https://www.home-assistant.io/integrations/tado/

### Version

2024.10.4

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by TASSDevon**
X devices are not supported, added a note to reflect this

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #35508

## Checklist

- [ x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **Documentation**
	- Added a note clarifying that new Tado X devices are not supported by the current integration and must be used through the Matter integration.

### CODE REVIEW

File: source/_integrations/tado.markdown [hunk 1]
```diff
@@ -38,6 +38,8 @@ There is currently support for the following device types within Home Assistant:
 - Sensor - for some additional information of the zones.
 - Weather - for information about the current weather at the location of your Tado home.
```
Comment (c0ffeeca7) [inline source/_integrations/tado.markdown:40]:
```suggestion
## Unsupported device types

```
I think this note is worth its own header. Could you also add a title ## Supported device types in line 31? I would make a suggestion but it too far out so I'm not allowed.

File: source/_integrations/tado.markdown [hunk 2]
```diff
@@ -38,6 +38,8 @@ There is currently support for the following device types within Home Assistant:
...
 
+Note: new Tado X devices are NOT supported by this integration, they have to be used through the Matter integration.
```
Comment (c0ffeeca7) [inline source/_integrations/tado.markdown]:
```suggestion
New Tado X devices are not supported by this integration, they have to be used through the [Matter integration](/integrations/matter).
```

File: source/_integrations/tado.markdown [hunk 3]
```diff
@@ -37,6 +37,9 @@ There is currently support for the following device types within Home Assistant:
 - [Presence detection](#presence-detection)
 - Sensor - for some additional information of the zones.
 - Weather - for information about the current weather at the location of your Tado home.
+## Unsupported device types
```
Comment (frenck) [inline source/_integrations/tado.markdown:41]:
```suggestion

## Unsupported device types
```

**Review by frenck — APPROVED** [review]
Thanks, @TASSDevon 👍 

../Frenck

**Pull request merged**